examples/single_project/README.md

```diff
@@ -0,0 +1,61 @@
+This repo.yaml file contains a workflow specification to use Infracost with a single project. It uses `infracost comment` to post Infracost results to your PR. 
+
+## Running with GitHub
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) dockerhub image
+2. You'll need to pass the following custom env vars into the container
+   ```sh
+   GITHUB_TOKEN=<your-github-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+       pre_workflow_hooks:
+         # Clean up any files left over from the last run
+         - run: rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json
```

Comment:
Are we sure all these envs are defined at the time of pre-workflow-hook run?  Seems like WORKSPACE & REPO_REL_DIR might be empty so maybe this should be moved to the plan: steps:?  If it's needed at all?